### PR TITLE
Update docs for mc admin update

### PR DESCRIPTION
**Pull request by nitisht**

### CODE REVIEW

File: docs/minio-admin-complete-guide.md [hunk 1]
```diff
@@ -289,18 +289,33 @@ Skip SSL certificate verification.
...
+
+*Example: Update all MinIO servers using custom download server.*
+```
+mc admin update myminio http://localhost:8000/release_file
+Server `play` updated successfully from RELEASE.2019-08-14T20-49-49Z to RELEASE.2019-08-21T19-59-10Z
+```
+
+> NOTE:
+> - An alias pointing to a distributed setup this command will automatically update all MinIO servers in the cluster.
+> - `update` is an disruptive operations for your MinIO service, any on-going API operations will be forcibly canceled. So, it should be used only when you are planning MinIO upgrades for your deployment.
```
Comment (kannappanr) [inline docs/minio-admin-complete-guide.md]:
```suggestion
> - `update` is a disruptive operation for your MinIO service, any on-going API operations will be forcibly canceled. So, it should be used only when you are planning MinIO upgrades for your deployment.
```

File: docs/minio-admin-complete-guide.md [hunk 2]
```diff
@@ -289,18 +289,33 @@ Skip SSL certificate verification.
...
 
+### Steps to update MinIO using a custom download server
+You can use any webserver as the download server for `mc` update. For `update` using custom download server for MinIO binary, you need to have two files:
+
+- Release information file in a format like this [http://dl.minio.io/server/minio/release/linux-amd64/minio.sha256sum]
+- Actual MinIO binary to be downloaded. This file should be named as `minio` for Linux environments and `minio.exe` for Windows environments.
+
+For example, if your custom download server is running at `localhost:8000`, and the release information file is available as `localhost:8000/release_info`, the MinIO binary should be available at `localhost:8000/minio`.
+
+*Example: Update all MinIO servers using custom download server.*
```
Comment (harshavardhana) [inline docs/minio-admin-complete-guide.md]: https://github.com/minio/minio/pull/9111#issuecomment-596988656

Some explanation on why this is not correct and we should mention how to do a centralized repo for all platforms.

**Comment by harshavardhana** [comment]
Something like this https://github.com/minio/mc/issues/3028#issuecomment-570125536

File: docs/minio-admin-complete-guide.md (hunk 2 above)
Comment (nitisht) [reply docs/minio-admin-complete-guide.md]: Updated the PR, PTAL

**Pull request merged**